publisher/publisher.go

```diff
@@ -483,16 +483,12 @@ func uploadRpm(conf config, srcTemplate string, upload Upload, arch string) (err
 			_ = os.Remove(signaturePath)
 		}
 
-		// check for .repo file and create if needed
-		if _, err = os.Stat(s3DotRepoFilepath); conf.accessPointHost != "" && os.IsNotExist(err) {
-			l.Println(fmt.Sprintf("creating 'newrelic-infra.repo' file in %s", s3RepoPath))
-
-			repoFileContent := generateRepoFileContent(conf.accessPointHost, destPath)
-
-			err := ioutil.WriteFile(s3DotRepoFilepath, []byte(repoFileContent), 0644)
-			if err != nil {
-				return err
-			}
+		// create .repo file
+		l.Println(fmt.Sprintf("creating 'newrelic-infra.repo' file in %s", s3RepoPath))
```

Comment:
Don't forget about this piece `conf.accessPointHost != ""`

Reply:
I thought about it but cannot be empty anymore. I can still leave the `if` if you think is safer :)

Reply:
good, if empty, then default, right?

Reply:
yes, fallback to prod